ethernet_init/Kconfig.projbuild

```diff
@@ -64,10 +64,9 @@ menu "Ethernet Configuration"
                     Goto https://www.microchip.com for more information about them.
 
             config ETHERNET_PHY_LAN867X
-                # bool "LAN867x"
+                bool "LAN867x"
                 # TODO: un-hide this option once processing of Matches and Rules is fixed in  IDF Component Manager and so LAN867x component can be conditionally included. 
```

Comment:
You un-hided it but you kept the `TODO`. Has been the component manager fixed yet?

Reply:
It's am, admittedly hacky, solution for hiding warnings. It is hidden as a result of `depends on !ETHERNET_INTERNAL_SUPPORT`.

The option is hidden and not included in the sdkconfig if internal EMAC support is enabled (making it disabled), and doesn't include it in the sdkconfig if internal EMAC support is disabled since the entire selection is omitted from sdkconfig.

It is fine as a temporary solution in order to prevent resulting warnings from having promptless choice.

Reply:
OK, fair enough but `TODO` is no more related.

Reply:
The `TODO` is still relevant because the option remains hidden and when I last checked the issue wasn't resolved yet.

Reply:
Now, its  `bool "LAN867x"` so it's not hidden. Maybe it's conditionally hidden. But that's something different.

Reply:
But with this condition it will never show up, thus making it effectively hidden. Rewording to 
```
# TODO: remove "depends on" once processing of Matches and Rules is fixed in  IDF Component Manager and so LAN867x component can be conditionally included. 
```
Will only make it more confusing since here `un-hide` implies the purpose of `depends on`